internal/pkg/service/appsproxy/proxy/pagewriter/error.go

```diff
@@ -0,0 +1,118 @@
+package pagewriter
+
+import (
+	"net"
+	"net/http"
+	"strings"
+
+	svcerrors "github.com/keboola/keboola-as-code/internal/pkg/service/common/errors"
+	"github.com/keboola/keboola-as-code/internal/pkg/utils/errors"
+)
+
+const ExceptionIDPrefix = "keboola-appsproxy-"
+
+type errorPageData struct {
```

Comment:
I would add a field "AppName", because we will need that for each page (selector,spinner,error)

Reply:
:heavy_check_mark: :hourglass:

Reply:
@hosekpeter I create two new PRs to address it:
- https://github.com/keboola/keboola-as-code/pull/1709
- https://github.com/keboola/keboola-as-code/pull/1710

I also added `projectId`, so we don't have to adjust it in some other PR